pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTenants.java

```diff
@@ -112,4 +112,16 @@ public CmdTenants(PulsarAdmin admin) {
         jcommander.addCommand("delete", new Delete());
     }
 
+    @Parameters(hidden = true)
+    static class CmdProperties extends CmdTenants {
+        public CmdProperties(PulsarAdmin admin) {
+            super(admin);
+        }
+
+        @Override
+        public boolean run(String[] args) {
+            System.err.println("WARN: The properties subcommand is deprecated. Please use tenants instead");
```

Comment:
The warning could break scripts if doing `pulsar-admin properties list | xargs ... `

Reply:
Shouldn't do unless they do 2>&1, since it's going to stderr